LPD-46765 - Close Filter Modal When a Applying Changes to the Filter

kresimir-coko opened the pull request.

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-10.liferay.com/job/test-portal-source-format/5975/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-10.liferay.com/userContent/jobs/test-portal-source-format/builds/5975/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 6 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/a99d6c114f02df79e8e66a85c94ee7ec6610ef89">a99d6c114f02df79e8e66a85c94ee7ec6610ef89</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/kresimir-coko/liferay-portal/tree/close-filter-modal">close-filter-modal</a><br/>Branch GIT ID: <a href="https://github.com/kresimir-coko/liferay-portal/commit/a3d48ec79dabb1621daec12525e6e6a0f67ecf88">a3d48ec79dabb1621daec12525e6e6a0f67ecf88</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-10.liferay.com/job/test-portal-source-format/5975/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-10.liferay.com/userContent/jobs/test-portal-source-format/builds/5975/jenkins-report.html">here</a>.</h5></details></html>

kresimir-coko commented:
ci:test:relevant

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 20 out of 20 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 24 out of 26 jobs passed in 1 hour 8 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/a99d6c114f02df79e8e66a85c94ee7ec6610ef89">a99d6c114f02df79e8e66a85c94ee7ec6610ef89</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/540cfe135a40ce0e528d518c730ac5ab4bdc6df2">540cfe135a40ce0e528d518c730ac5ab4bdc6df2</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/167898732">EE Development Acceptance (master) - 1246 - 2025-03-18[00:34:18]</a></p><details><summary><strong>ci:test:stable - 20 out of 20 jobs PASSED</strong></summary><details><summary><strong>20 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 23 out of 26 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest(master)/11138/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>central-requirements<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1205776/">central-requirements/0/0</a></li></ul></li><li>functional-smoke-tomcat90-postgresql155_stable<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1113183/">functional-smoke-tomcat90-postgresql155_stable/0/0</a></li></ul></li><li>js-unit<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1219336/">js-unit/0/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1352171/">js-unit/1/0</a></li></ul></li><li>modules-compile<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1205783/">modules-compile/0/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1113187/">modules-compile/0/1</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1352183/">modules-compile/0/2</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1304825/">modules-compile/0/3</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1106760/">modules-compile/0/4</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1205781/">modules-compile/0/5</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1113185/">modules-compile/0/6</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1219343/">modules-compile/0/7</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1205782/">modules-compile/0/8</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1352182/">modules-compile/0/9</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1352181/">modules-compile/0/10</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1106761/">modules-compile/0/11</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1113186/">modules-compile/0/12</a></li></ul></li><li>modules-integration-postgresql155_stable<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1205780/">modules-integration-postgresql155_stable/0/0</a></li></ul></li><li>modules-semantic-versioning<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1113176/">modules-semantic-versioning/0/0</a></li></ul></li><li>modules-unit_stable<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1106759/">modules-unit_stable/0/0</a></li></ul></li><li>playwright-js-tomcat90-mysql57<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1219345/">playwright-js-tomcat90-mysql57/0/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1113184/">playwright-js-tomcat90-mysql57/1/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1352180/"><strike>playwright-js-tomcat90-mysql57/2/0</strike></a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1352179/"><strike>playwright-js-tomcat90-mysql57/2/1</strike></a></li></ul></li><li>unit_stable<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1219344/">unit_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>23 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/11138/jenkins-report.html">here</a>.</h5><hr/><h4>This pull contains no unique failures.</h4><hr/><details><summary><strong>Failures in common with <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at 540cfe135a40ce0e528d518c730ac5ab4bdc6df2:</strong></summary><ol><li><div><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1352180//consoleText">playwright-js-tomcat90-mysql57/2/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-03/test-1-29/test-portal-acceptance-pullrequest(master)/11138/playwright-js-tomcat90-mysql57/2/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1352180/testReport/frontend-data-set-admin-web_tests_data-set-admin_creationActions.spec/ts">frontend-data-set-admin-web/tests/data-set-admin/creationActions.spec.ts</a></summary><div><ol><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1352180//testReport/frontend-data-set-admin-web_tests_data-set-admin_creationActions.spec/ts/Check_interactive_options_in_creation_action_form">Check interactive options in creation action form</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1352180/testReport/frontend-data-set-admin-web_tests_data-set-admin_itemActions.spec/ts">frontend-data-set-admin-web/tests/data-set-admin/itemActions.spec.ts</a></summary><div><ol><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1352180//testReport/frontend-data-set-admin-web_tests_data-set-admin_itemActions.spec/ts/Check_interactive_options_in_item_action_form">Check interactive options in item action form</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1352180/testReport/frontend-data-set-admin-web_tests_data-set-admin_customDataSets.spec/ts">frontend-data-set-admin-web/tests/data-set-admin/customDataSets.spec.ts</a></summary><div><ol><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1352180//testReport/frontend-data-set-admin-web_tests_data-set-admin_customDataSets.spec/ts/Assert_endpoint_with_resolved_paramater_is_available_as_an_option">Assert endpoint with resolved paramater is available as an option</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1352179//consoleText">playwright-js-tomcat90-mysql57/2/1</a> - <a href="https://storage.cloud.google.com/testray-results/2025-03/test-1-29/test-portal-acceptance-pullrequest(master)/11138/playwright-js-tomcat90-mysql57/2/1/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1352179/testReport/frontend-data-set-admin-web_tests_data-set-admin_systemDataSets.spec/ts">frontend-data-set-admin-web/tests/data-set-admin/systemDataSets.spec.ts</a></summary><div><ol><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1352179//testReport/frontend-data-set-admin-web_tests_data-set-admin_systemDataSets.spec/ts/Import_a_system_data_set_to_customize">Import a system data set to customize</a></li></ol></div></details></li></ol></div></li></ol></details></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/11138/bundles/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a></li></ul>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest(master)/11138/">test-portal-acceptance-pullrequest(master)#11138</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/11138/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/4800">liferay-frontend#4800</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/168112402">[master] ci:test:relevant - kresimir-coko &gt; liferay-frontend - PR#4800 - 2025-03-18[05:37:00]</a><br/></div><code><strong>Testray Build ID:</strong><span></span><code>168112402</code><br/></code><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-29.liferay.com/job/publish-testray-report/24611/">publish-testray-report#24611</a><br/></div></div>

juanjofgliferay commented:
@kresimir-coko There seems to be lots of commits on top of yours that don't belong to this branch. Would you mind rebasing?

kresimir-coko commented:
I did before pushing yesterday but I guess something went wrong 🤷 here it is again

juanjofgliferay commented:
@kresimir-coko Changes look good, just a couple details that might be considered.

The main filter component closes after adding the filter but, there is another filter dropdown (FilterResume) that behaves in a similar way and does not close after applying the filter.

https://github.com/user-attachments/assets/07274399-e00b-4713-9e24-cc55aed6a6fb

In FDS fragment tests we are manually closing filters dropdown (see x.e. [selectionFilters.spec.ts#L239](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-fragment/selectionFilters.spec.ts#L239)). With your changes this is not needed anymore.

kresimir-coko commented:
Sounds good @juanjofgliferay , gonna remove that from the tests and add the same functionality to the `FilterResume`

markocikos reviewed (commented):
@kresimir-coko The code LGTM! Do we need a test for this?

kresimir-coko commented:
IDK @markocikos I guess I can add it since I'm touching the files arealdy

markocikos commented:
@kresimir-coko In general, rule of the thumb is:
- If it's LPP, there must be a test
- If it's a Story, there must be a test
- If it's a technical "Task", or some other unusual type of a task, it depends. For some it doesn't make sense to have tests, for some it does. If no UI, it may be integration or unit test and not PW. It's best to ask in daily or slack :man_shrugging:

kresimir-coko commented:
I've modified the existing tests by removing the manual closing and just replaced it with an assertion that the submit filter button is hidden. I feel like that's enough to enhance the tests while not introducing much complexity as it's not an LPP.

I think this is done.

markocikos reviewed (approved):
@kresimir-coko LGTM! :sparkles:

kresimir-coko commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

The pull request will automatically be forwarded to the user `brianchandotcom` If the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-9.liferay.com/job/test-portal-source-format/6383/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-9.liferay.com/userContent/jobs/test-portal-source-format/builds/6383/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 6 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/88890198291b31a2e58bd67f5e1a6b398183f5bb">88890198291b31a2e58bd67f5e1a6b398183f5bb</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/kresimir-coko/liferay-portal/tree/close-filter-modal">close-filter-modal</a><br/>Branch GIT ID: <a href="https://github.com/kresimir-coko/liferay-portal/commit/f8993462e0030abdd165310d9c9c54942f049611">f8993462e0030abdd165310d9c9c54942f049611</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-9.liferay.com/job/test-portal-source-format/6383/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-9.liferay.com/userContent/jobs/test-portal-source-format/builds/6383/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest(master)/6534/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6534/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:stable - 8 out of 9 jobs passed</h3><h3>:x: ci:test:relevant - 9 out of 11 jobs passed in 1 hour 16 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/1258">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1327401_6534</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/88890198291b31a2e58bd67f5e1a6b398183f5bb">88890198291b31a2e58bd67f5e1a6b398183f5bb</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/e006823d87a3cdbb40c0571f3cf02f2684e47bb5">e006823d87a3cdbb40c0571f3cf02f2684e47bb5</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/171641369">EE Development Acceptance (master) - 1257 - 2025-03-21[04:34:17]</a></p><details><summary><strong>ci:test:stable - 8 out of 9 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li>central-requirements<ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/532672/">central-requirements/0/0</a></li></ul></li><li>functional-smoke-tomcat90-postgresql155_stable<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1033001/">functional-smoke-tomcat90-postgresql155_stable/0/0</a></li></ul></li><li>js-unit<ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/532673/"><strike>js-unit/0/0</strike></a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/502992/">js-unit/1/0</a></li></ul></li><li>modules-compile<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1222858/">modules-compile/0/0</a></li></ul></li><li>modules-integration-postgresql155_stable<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1208773/">modules-integration-postgresql155_stable/0/0</a></li></ul></li><li>modules-unit_stable<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1356282/">modules-unit_stable/0/0</a></li></ul></li><li>playwright-js-tomcat90-postgresql155_stable<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1033003/">playwright-js-tomcat90-postgresql155_stable/0/0</a></li></ul></li><li>unit_stable<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1033002/">unit_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>8 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 9 out of 11 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest(master)/6534/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>central-requirements<ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/532672/">central-requirements/0/0</a></li></ul></li><li>functional-smoke-tomcat90-postgresql155_stable<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1033001/">functional-smoke-tomcat90-postgresql155_stable/0/0</a></li></ul></li><li>js-unit<ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/532673/"><strike>js-unit/0/0</strike></a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/502992/">js-unit/1/0</a></li></ul></li><li>modules-compile<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1222858/">modules-compile/0/0</a></li></ul></li><li>modules-integration-postgresql155_stable<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1208773/">modules-integration-postgresql155_stable/0/0</a></li></ul></li><li>modules-semantic-versioning<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1038613/">modules-semantic-versioning/0/0</a></li></ul></li><li>modules-unit_stable<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1356282/">modules-unit_stable/0/0</a></li></ul></li><li>playwright-js-tomcat90-postgresql155_stable<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1033003/">playwright-js-tomcat90-postgresql155_stable/0/0</a></li></ul></li><li>unit_stable<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1033002/">unit_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>9 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6534/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/532673//consoleText">js-unit/0/0</a><pre>     [exec] PASS src/main/js/shared/components/__tests__/FilterAndOrder.jsx (9.341 s)
     [exec] PASS src/main/js/settings/data-privacy/hocs/__tests__/SuppressedUserList.jsx (5.196 s)
     [exec] PASS src/main/js/individual/dashboard/components/__tests__/TypeTrend.jsx (14.031 s)
     [exec] PASS src/main/js/settings/pages/user/__tests__/index.jsx (8.027 s)
     [exec] PASS src/main/js/settings/pages/__tests__/Workspace.jsx (17.876 s)
     [exec] PASS src/main/js/sites/hocs/mappers/__tests__/composition-query.js
     [exec] PASS src/main/js/shared/util/__tests__/recharts.jsx
     [exec] PASS src/main/js/segment/segment-editor/dynamic/inputs/components/__tests__/CustomSelectEntityInput.jsx (6.279 s)
     [exec] PASS src/main/js/shared/actions/__tests__/data-sources.js
     [exec] PASS src/main/js/settings/recommendations/components/__tests__/OutputVersionsCard.jsx (17.127 s)
     [exec] PASS src/main/js/event-analysis/components/event-analysis-editor/event-analysis-builder/__tests__/EventAnalysisBuilder.jsx
     [exec] PASS src/main/js/event-analysis/components/event-analysis-editor/event-analysis-builder/base-dropdown/__tests__/ListItem.jsx
     [exec] PASS src/main/js/shared/components/__tests__/Breadcrumbs.jsx
     [echo] 
&lt;?xml version="1.0" encoding="UTF-8"?&gt;
&lt;exec dir="modules" executable="/opt/dev/projects/github/liferay-portal/gradlew" failonerror="true"&gt;
  &lt;arg value="--continue" /&gt;
  &lt;arg value="--max-workers=4" /&gt;
  &lt;arg value="--no-daemon" /&gt;
  &lt;arg value="--profile" /&gt;
  &lt;arg value="--project-cache-dir=/opt/dev/projects/github/liferay-portal/.gradle" /&gt;
  &lt;arg value="--stacktrace" /&gt;
  &lt;arg value="-Dapp.server.parent.dir=/opt/dev/projects/github/liferay-portal/bundles" /&gt;
  &lt;arg value="-Dforced.cache.enabled=true" /&gt;
  &lt;arg value="-Dliferay.home=/opt/dev/projects/github/liferay-portal/bundles" /&gt;
  &lt;arg value="-Dorg.gradle.internal.http.connectionTimeout=120000" /&gt;
  &lt;arg value="-Dorg.gradle.internal.http.socketTimeout=120000" /&gt;
  &lt;arg line=":dxp:apps:osb:osb-faro:osb-faro-web:packageRunTest" /&gt;
&lt;/exec&gt;
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-common.xml:1033: Unable to execute Gradle task: :dxp:apps:osb:osb-faro:osb-faro-web:packageRunTest.
[stopwatch] [run.batch.test.action: 8:53.942 sec]
     [echo] 1.0 JS unit tests failed.
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null1051288736.properties</pre><ol><li><details><summary><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/532673/testReport/com.liferay.jenkins/JenkinsLogAsserterTest">com.liferay.jenkins.JenkinsLogAsserterTest</a></summary><div><ol><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/532673//testReport/com.liferay.jenkins/JenkinsLogAsserterTest/testScanJenkinsLog">testScanJenkinsLog</a></li></ol></div></details></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6534/bundles/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a></li></ul>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest(master)/6534/">test-portal-acceptance-pullrequest(master)#6534</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6534/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/4800">liferay-frontend#4800</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/171748602">[master] ci:test:relevant - kresimir-coko &gt; liferay-frontend - PR#4800 - 2025-03-21[10:00:15]</a><br/></div><code><strong>Testray Build ID:</strong><span></span><code>171748602</code><br/></code><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-32.liferay.com/job/publish-testray-report/37120/">publish-testray-report#37120</a><br/></div></div>

kresimir-coko commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

The pull request will automatically be forwarded to the user `brianchandotcom` If the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
Skipping previously passed test suites:
- `ci:test:sf`

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/21227/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/21227/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
All required test suite(s) passed.
Forwarding pull request to `brianchandotcom`.
[Console](https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/21227//consoleFull)

liferay-continuous-integration commented:
Pull request has been successfully forwarded to  https://github.com/brianchandotcom/liferay-portal/pull/160210
[Console](https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/21227//consoleFull)

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/21227/">test-portal-acceptance-pullrequest(master)#21227</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/21227/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/4800">liferay-frontend#4800</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/173186245">[master] ci:test:relevant - kresimir-coko &gt; liferay-frontend - PR#4800 - 2025-03-24[02:22:01]</a><br/></div><code><strong>Testray Build ID:</strong><span></span><code>173186245</code><br/></code><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-37.liferay.com/job/publish-testray-report/45663/">publish-testray-report#45663</a><br/></div></div>
